### PR TITLE
Added curl workaround when LWP module is not available

### PR DESCRIPTION
**Pull request by HavrilaJ**

### CODE REVIEW

**Comment by gwarf** [comment]
Thanks!
Should fix #13

File: pakiti-client [hunk 1]
```diff
@@ -369,9 +368,42 @@ sub encrypt_report ($) {
...
+    my($ua, $response);
+    my($code,$body);
+    eval { require LWP::UserAgent; LWP::UserAgent->import; };
+    if ($@) {
+	if ($Option{"disable-tls-checks"}) {
+            ($code,$body) = `curl -sik -X POST $url -d "$report"` =~m{\A\S+ (\d+) .*?\r?\n\r?\n(.*)}s;
+        } else {
+            ($code,$body) = `curl -si -X POST $url -d "$report"` =~m{\A\S+ (\d+) .*?\r?\n\r?\n(.*)}s;
+        }
+        if ($body =~ m/200 OK/) {
```
Comment (kouril) [inline pakiti-client]: this check seems potentially fragile. It'd be good to check the exec code of curl. For the check of HTTP code it'd be better to make sure you really check the correct HTTP header not just a random "200 OK" string in the message.

**Comment by gwarf** [comment]
Any news about this? It would be good to be able to update the version in CVMFS so that there won't be any issue with sites not having LWP installed.

**Comment by HavrilaJ** [comment]
I made a little fix, I hope it will be ok and we will check/merge it.

File: pakiti-client [hunk 2]
```diff
@@ -369,9 +368,41 @@ sub encrypt_report ($) {
...
+    my ($url, $report) = @_;
+    my($ua, $response);
+    eval { require LWP::UserAgent; LWP::UserAgent->import; };
+    if ($@) {
+        if ($Option{"disable-tls-checks"}) {
+            $response = `curl -sik -o /dev/null -w "%{http_code}" -X POST $url -d "$report"`;
+        } else {
+            $response = `curl -si -o /dev/null -w "%{http_code}" -X POST $url -d "$report"`;
+        }
+        if ($response == "200") {
```
Comment (kouril) [inline pakiti-client]: in order to maintain the readability, I'd stick with the same pattern as in the other if branch (and the original code), i.e., ($response == "200") or die

File: pakiti-client [hunk 3]
```diff
@@ -369,9 +368,41 @@ sub encrypt_report ($) {
...
+            print(STDERR "report successfully sent\n") if -t STDERR;
+            print "OK\n";
```
Comment (kouril) [inline pakiti-client]: why is the OK written here while not in the other branch?

File: pakiti-client (hunk 2 above)
Comment (kouril) [inline pakiti-client]: didn't you actually mean to check `http_code` (not `response`) here?

**Pull request merged**